config/config.go

```diff
@@ -25,21 +23,14 @@ func LoadConfiguration() *viper.Viper {
 	config.Token = v.GetString("CONSUL_TOKEN")
 
 	c, err := api.NewClient(config)
-	PanicOnError(err, "")
+	utils.PanicOnError(err, "")
 
 	pair, _, err := c.KV().Get(v.GetString("CONSUL_PATH"), nil)
-	PanicOnError(err, "Invalid consul config.")
+	utils.PanicOnError(err, "Invalid consul config.")
```

Comment:
It acts the same way as `panic` - logs to stderr - which can be redirected. Intended for use failing-fast on bad config or to aid development printing err messages before working out better ways to handle them. Appropriate use of `panic` would apply to this.
So golang core has panic instead of this 2 liner:
```
fmt.Fprintln(os.Stderr, err)
os.Exit(1)
```

I'm just taking it one step further by proposing `PanicOnError` instead of this 3 liner:
```
if err != nil {
  panic(msg + "ERROR: " + err.String())
}
```
Yeah wouldn't suggest using this for recoverable errors.